Cargo.toml

```diff
@@ -23,6 +23,7 @@ rand = { version = "0.8", optional = true }
 uuid = { version = "1.1", optional = true }
 postgres-types = { version = "0.2.6", optional = true }
 bytes = { version = "1.4.0", optional = true }
+bitcode = { version = "0", features = ["derive"], default-features = false, optional = true }
```

Comment:
This should specify the minor version for major=0 versions.

Reply:
Specifying `version = "0"` allows the user to pick the major version of bitcode. Since ulid has reached version 1, it can't make breaking changes to update bitcode. For example if ulid added support for `bitcode = "0.6"` and `bitcode = "0.7"` was released, ulid couldn't update to it because that would be a breaking change.

`bitcode = "0"` allows any version the user wants to pick such as `0.5`, `0.6` and future `0.7`. The only tricky part is making sure ulid doesn't use APIs that will break between major versions. Luckily, [bitcode docs](https://docs.rs/bitcode/0.6.0/bitcode/#library-example) specify that `#[derive(bitcode::Encode, bitcode::Decode)]` and `feature = "derive"` won't ever change between major versions.

While this method is unconventional, it allows 3rd party crates to support bitcode while allowing bitcode to remain unstable. One of bitcode's core goals is to keep improving its binary format. Unfortunately format changes are breaking changes so bitcode can't ever reach `1.0`.